tests/csm/rest/test_csm_load.py

```diff
@@ -137,6 +139,16 @@ def teardown_method(self):
         for iam in iam_deleted:
             self.iam_users_created.remove(iam)
 
+        for csm_user in self.csm_users_created:
+            resp = self.csm_obj.delete_csm_user(csm_user)
+            if resp:
+                csm_deleted.append(csm_user)
+            else:
+                self.log.error("IAM deletion failed for %s ", csm_user)
+        self.log.info("CSM user deleted %s", csm_deleted)
+        for csm in csm_deleted:
+            self.csm_users_created.remove(csm)
```

Comment:
This for loop is not required. It can be take care in the if resp line 145

Reply:
done